src/js/board.js

```diff
@@ -0,0 +1,31 @@
+const lettersId = "ABCDEJGH";
+
+export default class Board {
+  init() {
+    const app = document.getElementById("app");
+    if (!app){
+      console.error("App wasn't found");
+      return;
+    }
```

Comment:
You can omit the return by using the Error class. Like so:

```suggestion
    if (!app) throw new Error("App wasn't found")
```